pkg/lwip/contrib/netdev2/lwip_netdev2.c

```diff
@@ -47,7 +47,6 @@
 #define ETHERNET_IFNAME2 'T'
 
 /* running number for different interfaces */
-static uint8_t _num = 0;
```

Comment:
I stated this already in the commit message, but something like this already happens lwIP internally so no need for this and potentially makes things just more confusing ;-)